packages/config/eslint/index.js

```diff
@@ -16,5 +16,15 @@ module.exports = {
   rules: {
     "@next/next/no-html-link-for-pages": "off",
     "react/jsx-key": "off",
+    "@typescript-eslint/consistent-type-imports": "warn",
+    "@typescript-eslint/no-unused-vars": [
```

Comment:
these 2 rules are pretty nice, feel free to remove them if you dont like it

Reply:
Nice! I'm not too familiar with eslint rules but do like them very much so I appreciate it. 🙌 

PS: I actually thought the latter was included with `eslint:recommended`, which I had seen earlier somewhere. But that works in any case.

Reply:
no_unused_vars is included, but I've also added an ignore pattern so you can prepend the variable with `_` to get it ignored - which is not part of the recommended rule